aries_vcx/src/utils/mockdata/mockdata_connection.rs

```diff
@@ -1,409 +1,543 @@
-// Alice receives invitation via out of band channel
-pub const ARIES_CONNECTION_INVITATION: &str = r#"
-{
-    "@id": "28b39b79-f5db-4478-879a-15bb12632d00",
-    "@type": "https://didcomm.org/connections/1.0/invitation",
-    "label": "alice-e9b498a1-7d86-4389-a9de-3823dbb2f27e",
-    "recipientKeys": [
-        "DEKbrMDX9LBGhCk4LBhH6t5B6Kh5iE7GvfepAJYXp7GX"
-    ],
-    "routingKeys": [
-        "C9JGq5BLcZNAQZ3x27w9cHTA7N6dysZThjLjjPRbvDoC",
-        "Hezce2UWMZ3wUhVkh2LfKSs8nDzWwzs2Win7EzNN3YaR"
-    ],
-    "serviceEndpoint": "http://localhost:8080/agency/msg"
-}"#;
-
-// Alice created and serialized connection created from received invitation
 pub const CONNECTION_SM_INVITEE_INVITED: &str = r#"
 {
-    "version": "1.0",
-    "source_id": "alice-670c6360-5c0e-4495-bd25-2ee58c39fc7e",
-    "thread_id": "b5517062-303f-4267-9a29-09bc89497c06",
```

Comment:
Although I see `thread_id`and `source_id` defined `SmConnectionInvitee`, this information seems to be not included in serialized Connection. I think we should, especially the `thread_id`.
I am not fan of the existence of `source_id`, but given it's everywhere else, probably it should be included equally here as well. 
We could perhaps remove `source_id` across the board in separate PR in the future.